controller/src/beerocks/bml/bml.cpp

```diff
@@ -556,3 +556,77 @@ int bml_channel_selection(BML_CTX ctx, const char *al_mac, const char *ruid)
 
     return (pBML->channel_selection(al_mac, ruid));
 }
+
+int bml_set_dcs_continuous_scan_enable(BML_CTX ctx, const char *radio_mac, int enable)
+{
+    // Validate input parameters
+    if (!ctx) {
+        return (-BML_RET_INVALID_ARGS);
+	}
+
+	// TODO: call suit bml api
+	return 0;
+}
+
+int bml_get_dcs_continuous_scan_enable(BML_CTX ctx, const char *radio_mac, int *output_enable)
+{
+    // Validate input parameters
+    if (!ctx) {
+        return (-BML_RET_INVALID_ARGS);
+	}
+
+	// TODO: call suit bml api
+	return 0;
+}
+
+int bml_set_dcs_continuous_scan_params(BML_CTX ctx, const char *radio_mac, int dwell_time,
+                                       int interval_time, unsigned int *channel_pool,
+                                       int channel_pool_size)
+{
+    // Validate input parameters
+    if (!ctx) {
+        return (-BML_RET_INVALID_ARGS);
+	}
+
+	// TODO: call suit bml api
+	return 0;
```

Comment:
I think that we should return failure until these stubs are implemented. This goes to all the stubs below and above this one. and also add LOG.

Reply:
This MUST return failure, I don't care if @itayx will change it soon - cause until he does, the API should not report success when it's not implemented..